cmd/nodepool/kubevirt/create.go

```diff
@@ -112,6 +115,30 @@ func (o *RawKubevirtPlatformCreateOptions) Validate() (*ValidatedKubevirtPlatfor
 	if len(o.AdditionalNetworks) == 0 && o.AttachDefaultNetwork != nil && !*o.AttachDefaultNetwork {
 		return nil, fmt.Errorf(`missing --additional-network. when --attach-default-network is false configuring an additional network is mandatory`)
 	}
+
+	for _, hostDevice := range o.HostDevices {
+		split := strings.Split(hostDevice, ",")
+		if len(split) > 2 {
+			return nil, fmt.Errorf("invalid host device name parameter [%s]", hostDevice)
+		}
+		if len(split) == 2 && !strings.HasPrefix(split[1], "count:") {
+			return nil, fmt.Errorf("invalid host device count format [%s]. 'count:' is missing", hostDevice)
+		}
+		if len(split) == 2 && strings.HasPrefix(split[1], "count:") {
+			countSplit := strings.Split(split[1], ":")
+			if len(countSplit) != 2 {
+				return nil, fmt.Errorf("invalid host device count format [%s]. too many colons", hostDevice)
+			}
+			count, err := strconv.Atoi(countSplit[1])
+			if err != nil {
+				return nil, fmt.Errorf("invalid host device count format. unable to parse %s as integer: %w", countSplit[1], err)
+			}
+			if count < 1 {
+				return nil, fmt.Errorf("host device count must be greater than or equal to 1. received: %d", count)
+			}
+		}
+	}
+
```

Comment:
there's a new unit test for this create command `cmd/cluster/kubevirt/create_test.go`

I think it would be good to exercise this host device cli arg there.

Reply:
a bunch of unit tests have been added for this CLI arg.